app/facades/foods_facade.rb

```diff
@@ -0,0 +1,10 @@
+class FoodsFacade
+
+  def self.search_by_food(food)
+  	foods_data = FoodsService.foods_by_ingredient(food)[:foods][0..9]
```

Comment:
Can you find another way to limit the size to 10 without needing to use this ruby logic? (hint: API docs!)